findex_in_memory/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
if you don't mind, we could move folders `findex_in_memory` and `findex_sqlite` to the root folder `examples` (in order to be build via `cargo build --examples`)

Reply:
I tried several possibilities but the only one allowing me to do what I wanted is the present one...

What I want is to be able to import structures defined in these crates for tests and benchmarks (at least). This is not possible (at least I couldn't get it working) by putting them in the `tests` directory. I bet it would be the same with the `examples` directory.